components/Navbar.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box } from '@mui/material';
+import Link from 'next/link';
+import { FC } from 'react';
```

Comment:
when importing type, you should use 
```
import type { FC } from 'react';
```
useful for brevity.

please update all your import statement **accordingly**.

Reply:
done